addons/database-backup-and-maintenance.sh

```diff
@@ -95,13 +91,10 @@ if [ -f /var/run/mysqld/mysqld.pid ]; then
     if [ `/bin/date +%d` -eq '01' ]; then
```

Comment:
This line has given us trouble in the past (on Jan. 1st for example).
It would be better if we ran this every night in smaller increments.